src/read.c

```diff
@@ -4538,6 +4566,16 @@ static avifResult avifDecoderDecodeTiles(avifDecoder * decoder, uint32_t nextIma
     return AVIF_RESULT_OK;
 }
 
+static avifBool avifAWholeFrameWasDecoded(const avifDecoder * decoder)
```

Comment:
Nit/optional: libavif often uses an object-oriented naming convention for functions. If a function's main parameter is `avifFoo`, then the function is often named `avifFooDoSomething`.

Under that naming convention, this function would be named something like `avifDecoderDataFullyDecoded` or `avifDecoderDataCompletelyDecoded`